clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -3355,6 +3355,7 @@ public DeleteRecordsResult deleteRecords(final Map<TopicPartition, RecordsToDele
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
The reason of  adding this suppress annotation is that the method still invoke the deprecated method ?
If so, we could replace the deprecated method with new one which doesn't not have typo and we don't need the suppress annotation.